kuma/wiki/jinja2/403-create-page.html

```diff
@@ -1,6 +1,6 @@
 {% extends "403.html" %}
 
-{% block page_title %}{{ _("You're trying to create a new page.") }}{% endblock %}
+{{ _("You're trying to create a new page.") }}
```

Comment:
The ``{% block page_title %}`` should remain, so that the ``{% extends "403.html" %}`` will work properly.